azure-pipelines.yml

```diff
@@ -150,12 +178,44 @@ stages:
           continueOnError: true
           condition: always()
 
+- ${{ if eq(variables._RunAsPublic, True) }}:
+  - stage: test
+    displayName: Run Integration Tests
+    dependsOn: Build_OSX
+    jobs:
+    - job: integration_tests
+      displayName: iOS Integration Tests
+      pool:
+        name: Hosted macOS
+      steps:
+      - download: current
+        artifact: Microsoft.DotNet.XHarness.CLI.OSX.IntegrationTests.Debug
+
+      - template: /eng/common/templates/steps/send-to-helix.yml
```

Comment:
This is good for now, but we should get this into the Arcade Repo and consume the Helix SDK XHarness support (if you give me a functioning app and the args for it I can drive that)

Reply:
I agree. As you say, for now it's ok since the value I am after is to verify that the machines I have are running XHarness fine